doc/source/index.rst

```diff
@@ -2,11 +2,9 @@
 The Python ARM Radar Toolkit - Py-ART
 =====================================
 
-Latest Release
-==============
-
 .. toctree::
-    :maxdepth: 4
+    :maxdepth: 1
```

Comment:
I think it's whining about this. Try setting it back to 4.

Reply:
Gotcha, will try that

Reply:
@rcjackson Dang, sadly did not work.

Reply:
If you try to access one of those functions in the docs can you still get to it? I just would like to know how important these warnings actually are, so this will tell me.

Reply:
Yeah it seems I can navigate through all the functions and their attributes and everything are there. One thing I notices is params show up as gate_yarray instead of gate_y: array so wondering if the old docs all colons need a space removed from before the colon

Reply:
In PyDDA I only put a space after the colon, not before. So that might be causing it to not work with this theme. I recall we ran into something similar with ACT.

Reply:
Gotcha, I'll go through and change all of that.